docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc

```diff
@@ -1,11 +1,21 @@
-[id='pro-running-the-published-repositories-{chapter}']
-= Introduction
+[id='{context}-pro-running-the-demo-repositories']
+= Running the Demo App from the Published Repositories
+
+The {WFM-RC-NameShort} demo applications in this section are dependent on the _public NPM Registry_ and, do not require compilation via the TypeScript compiler and linking via Lerna.js.
```

Comment:
+The {WFM-RC-NameShort} applications

Reply:
`do not require compilation via the TypeScript compiler and linking via Lerna.js.` - Let's remove this as this content is not needed and missleading for eventual reader.